src/drivers/vue.js

```diff
@@ -25,6 +25,16 @@ type VueType = {|
     component : (string, VueComponent) => RegisteredVueComponent
 |};
 
+function propsToCamelCase(props : Object) : Object {
+    for (const key of Object.keys(props)) {
```

Comment:
Can we avoid mutating the props object passed in and instead return a new object? For example, we could use `Array.prototype.reduce` to return a new altered copy: Ex:

```js
function propsToCamelCase(props : Object) : Object {
  return Object.keys(props).reduce((key, acc) => {
            // add logic here
  }, {})
}
```

Reply:
@gregjopa I have updated the logic to use reduce instead of mutating the object as suggested.